rust/src/lib.rs

```diff
@@ -260,20 +260,20 @@ pub mod http {
         }
         // 13. Print content
         let mut in_angle = false;
-        let mut out = String::new();
+        let mut out: Vec<u8> = Vec::new();
         let body = get_body(body);
         for c in body {
             match *c {
                 b'<' => in_angle = true,
                 b'>' => in_angle = false,
                 _ => {
                     if !in_angle {
-                        out.push(*c as char);
+                        out.push(*c);
                     }
                 }
             }
         }
-        out
+        String::from_utf8(out).unwrap()
```

Comment:
non-utf8 websites does frequently exist in real world. So `expect()` with message will be more useful than `unwrap()` here.